### PR TITLE
add dynamodb describe table

### PR DESCRIPTION
**Pull request by geronimo-iia**
Adding dynamodb describetable method. It's usefull to find LatestStreamArn when ressource are declared outside 'serverless'.

Thanks for your work !

### CODE REVIEW

**Comment by geronimo-iia** [comment]
@jacob-meacham if you could check this pull request, it will be very appreciated 
thanks again

File: src/index.js [hunk 1]
```diff
@@ -104,6 +104,19 @@ async function getKinesisValue(key, awsParameters) {
...
 }
 
+/**
+ * @param key the name of the DynamoDb table to resolve
+ * @param awsParameters parameters to pass to the AWS.DynamoDB constructor
+ * @returns {Promise<AWS.DynamoDB.Table>}
+ * @see http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB.html#describeTable-property
+ */
+async function getDynamoDbValue(key, awsParameters) {
+  winston.debug(`Resolving Kinesis stream with name ${key}`)
```
Comment (jacob-meacham) [inline src/index.js]: Typo in the logs: Kinesis -> Dynamo

File: src/index.js [hunk 2]
```diff
@@ -104,6 +104,19 @@ async function getKinesisValue(key, awsParameters) {
...
 }
 
+/**
+ * @param key the name of the DynamoDb table to resolve
+ * @param awsParameters parameters to pass to the AWS.DynamoDB constructor
+ * @returns {Promise<AWS.DynamoDB.Table>}
+ * @see http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB.html#describeTable-property
+ */
+async function getDynamoDbValue(key, awsParameters) {
+  winston.debug(`Resolving DynamoDB stream with name ${key}`)
```
Comment (geronimo-iia) [inline src/index.js]: @jacob-meacham I've doubted between dynamo or dynamoDB. Don't hesitate to revert to me if i should change it again.

**Comment by jacob-meacham** [comment]
Looks great, thanks for the change!

**Comment by jacob-meacham** [comment]
@geronimo-iia - released as 1.2.0

**Pull request merged**